src/content/api/_endpoints/promotions-create.js

```diff
@@ -4,6 +4,7 @@ export default {
   request: {
     headers: {
       'X-Api-Key': '<TOKEN>',
+      'Content-Type': 'application/json',
```

Comment:
@subarashii-huch09 Suggestion: Check if any of our other APIs have this issue. Fix any others too 😄

Reply:
@tom-blake  Yep, great point! 🙂 I'll check other APIs too. 🫡